dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/views/AbstractView.java

```diff
@@ -41,14 +41,22 @@ public AbstractView() {
 		heightProperty().addListener((o, v1, v2) -> toCenter.setY(v2.intValue() / 2));
 
 		translate = new Translate();
-		scale = getScale();
+		setScale(SCALE);
 	}
 
 	/**
+	 * Set the initial scale.
+	 * @param scale the scale applied to the {@link AbstractView}.
+	 */
+	public void setScale(double scale) {
+		this.scale = new Affine(new Scale(scale, scale));
+	}
+	
+	/**
 	 * @return The scale used to scale the contents of the {@link AbstractView}.
 	 */
 	public Affine getScale() {
-		return new Affine(new Scale(SCALE, SCALE));
```

Comment:
Not necessarily needed. I thought this would be more consistent with other getters, as it would return the actual currently used scale.